server/router.go

```diff
@@ -45,5 +45,10 @@ func Serve() {
 	srv.PUT("/devices/:d_id/sensors/:s_id/", updateSensor)
 	srv.DELETE("/devices/:d_id/sensors/:s_id/", deleteSensor)
 
+	// Routes for VALUES
```

Comment:
Please, remove the trailing slashes for resource endpoint routes.

For example, `srv.GET("/devices/", readDevices)` --> `srv.GET("/devices", readDevices)`